### PR TITLE
Fix failures in SseCrossOriginIT

### PR DESCRIPTION
**Pull request by cmebarrow**
This test frequently fails because the robot scripts are closing the backend and the client at once which can cause disconnected event on backend instead of closed because closing backend causes broadcast service to close client connections, leading to a race. Fixed  in the robot scripts by waiting for client connection to be closed before closing backend.

### CODE REVIEW

File: transport/sse/src/test/scripts/org/kaazing/gateway/transport/sse/sse.connect.and.get.data.with.newline.rpt [hunk 1]
```diff
@@ -61,6 +61,6 @@ read notify CONNECTED_AND_CAN_SEND_DATA_FROM_BACKEND_TO_FRONT
 read "data:Kobe is greater than Griffen\n"
 read "data:\n"
 read "\n"
-read notify TEST_OVER
 close
-closed
\ No newline at end of file
+closed
+read notify TEST_OVER
```
Comment (brennangaunce) [inline transport/sse/src/test/scripts/org/kaazing/gateway/transport/sse/sse.connect.and.get.data.with.newline.rpt]: Are there funny characters in here?

**Comment by brennangaunce** [comment]
LGTM.

**Comment by dpwspoon** [comment]
LGTM other than the new lines.  Ping me when its fixed and I'll merge it

**Comment by cmebarrow** [comment]
@dpwspoon Newlines fixed

**Pull request merged**